data/json/construction.json

```
@@ -7111,6 +7111,112 @@
+  },
+  {
+    "type": "construction",
+    "id": "constr_clear_rubble_rock",
+    "group": "clear_rubble",
+    "category": "OTHER",
+    "qualities": [ [ { "id": "DIG", "level": 1 } ] ],
+    "required_skills": [ [ "survival", 0 ] ],
+    "time": "1 m",
+    "byproducts": [
+      { "item": "rock", "count": [ 0, 2 ] },
+      { "item": "rock_flaking", "count": [ 0, 1 ] },
+      { "item": "flint", "count": [ 0, 1 ] },
+      { "item": "sharp_rock", "count": [ 0, 1 ] },
+      { "item": "pebble", "charges": [ 0, 4 ] }
+    ],
+    "pre_flags": "RUBBLE",
+    "pre_terrain": "f_rubble_rock",
+    "post_terrain": "f_null",
+    "activity_level": "EXTRA_EXERCISE"
+  },
+  {
+    "type": "construction",
+    "id": "constr_clear_rubble_landfill",
+    "group": "clear_rubble",
+    "category": "OTHER",
+    "qualities": [ [ { "id": "DIG", "level": 1 } ] ],
+    "required_skills": [ [ "survival", 0 ] ],
+    "time": "1 m",
+    "byproducts": [
+      { "item": "glass_shard", "count": [ 0, 4 ] },
+      { "item": "plastic_chunk", "count": [ 0, 1 ] },
+      { "item": "ceramic_shard", "count": [ 0, 1 ] },
+      { "item": "splinter", "count": [ 0, 1 ] },
+      { "item": "scrap_cotton", "count": [ 0, 1 ] }
+    ],
+    "pre_flags": "RUBBLE",
+    "pre_terrain": "f_rubble_landfill",
+    "post_terrain": "f_null",
+    "activity_level": "EXTRA_EXERCISE"
+  },
+  {
+    "type": "construction",
+    "id": "constr_clear_rubble_wreckage",
+    "group": "clear_rubble",
+    "category": "OTHER",
+    "qualities": [ [ { "id": "DIG", "level": 1 } ] ],
+    "required_skills": [ [ "survival", 0 ] ],
+    "time": "1 m",
+    "byproducts": [
+      { "item": "scrap", "charges": [ 0, 3 ] },
+      { "item": "slag", "charges": [ 0, 1 ] },
+      { "item": "scrap_tin", "count": [ 0, 1 ] },
+      { "item": "scrap_aluminum", "count": [ 0, 1 ] },
+      { "item": "scrap_copper", "count": [ 0, 1 ] }
+    ],
+    "pre_flags": "RUBBLE",
+    "pre_terrain": "f_wreckage",
+    "post_terrain": "f_null",
+    "activity_level": "EXTRA_EXERCISE"
+  },
+  {
+    "type": "construction",
+    "id": "constr_clear_rubble_woodchips",
+    "group": "clear_rubble",
+    "category": "OTHER",
+    "qualities": [ [ { "id": "DIG", "level": 1 } ] ],
+    "required_skills": [ [ "survival", 0 ] ],
+    "time": "1 m",
+    "byproducts": [ { "item": "splinter", "count": [ 0, 3 ] } ],
+    "pre_flags": "RUBBLE",
+    "pre_terrain": "f_woodchips",
+    "post_terrain": "f_null",
+    "activity_level": "EXTRA_EXERCISE"
+  },
+  {
+    "type": "construction",
+    "id": "constr_clear_rubble_ash",
+    "group": "clear_rubble",
+    "category": "OTHER",
+    "qualities": [ [ { "id": "DIG", "level": 1 } ] ],
+    "required_skills": [ [ "survival", 0 ] ],
+    "time": "1 m",
+    "byproducts": [ { "item": "ash", "charges": [ 0, 50 ] } ],
+    "pre_flags": "RUBBLE",
+    "pre_terrain": "f_ash",
+    "post_terrain": "f_null",
+    "activity_level": "EXTRA_EXERCISE"
+  },
+  {
     "id": "constr_bench_wooden",
```

```suggestion
  {
    "type": "construction",
    "id": "constr_bench_wooden",
```
Looks you lost this line when you merged master